### PR TITLE
Add test file

### PR DESCRIPTION
**Pull request by Vasilov345**
This is test PR to show my students GitHub power

### CODE REVIEW

File: file_for_changes.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Some text
```
Comment (Vasilov345) [inline file_for_changes.txt]: This is some comment

Comment (Galyopa) [inline file_for_changes.txt]: TEst

Comment (AlexUnderAchkasov) [inline file_for_changes.txt]: 0_о

Comment (Tarapatina) [inline file_for_changes.txt]: HI)

Comment (drglutton1) [inline file_for_changes.txt]: comment

**Comment by YaschukP** [comment]
Привет

File: file_for_changes.txt (hunk 1 above)
Comment (Vasilov345) [reply file_for_changes.txt]: This code is awesome. What do you want from me !?

Comment (Ahiligor) [reply file_for_changes.txt]: O_o

**Pull request merged**